packages/server/src/standalone/index.ts

```diff
@@ -0,0 +1,149 @@
+import { json } from 'body-parser';
+import cors from 'cors';
+import express from 'express';
+import http from 'http';
+import type { AddressInfo, ListenOptions } from 'net';
+import { format as urlFormat } from 'url';
+import { ApolloServer } from '../ApolloServer';
+import { ExpressContext, expressMiddleware } from '../express';
+import type { BaseContext, ContextFunction } from '../externalTypes';
+import type { WithRequired } from '../types';
+
+interface HTTPServerOptions<TContext extends BaseContext> {
+  context?: ContextFunction<[ExpressContext], TContext>;
+}
+
+export function httpServer(
+  server: ApolloServer<BaseContext>,
+  options?: {},
```

Comment:
in expressMiddleware we did
```
options?: ExpressMiddlewareOptions<BaseContext>
```
here which I think seems reasonable... I guess it technically lets you write a context function that does nothing, and means that if we add another option then we won't have to change the line.

Reply:
I think I'm more interested in being consistent with the pattern here. If we decide to do `{}` or something similar instead it'd be best to do in both places I think. Going to go with what the express pkg is doing.

Reply:
[f6b44fd](https://github.com/apollographql/apollo-server/pull/6420/commits/f6b44fd4e533e3abad21f0bd4029b816888a9bf3)